source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -100,7 +149,9 @@ RoleBasedAccessControlRouteSpecificFilterConfig::RoleBasedAccessControlRouteSpec
     const envoy::extensions::filters::http::rbac::v3::RBACPerRoute& per_route_config,
     Server::Configuration::ServerFactoryContext& context,
     ProtobufMessage::ValidationVisitor& validation_visitor)
-    : per_rule_stats_(per_route_config.rbac().track_per_rule_stats()) {
+    : rules_stat_prefix_(per_route_config.rbac().rules_stat_prefix()),
```

Comment:
per_route_config.rbac().rules_stat_prefix() will be empty if not specified right? just double checking to make sure this won't change existing behavior.

Reply:
Good catch, I fixed this to be backward compatible and added a test.